### ISSUE TITLE
Products displaying in shop that aren't in order cycle . . 

### ISSUE THREAD

**Issue by kirstenalarsen** [openfoodfoundation/openfoodnetwork#697]
order cycle is open - 
![image](https://cloud.githubusercontent.com/assets/2911176/10602340/6c6bba2e-7763-11e5-8083-54a940e44839.png)

does not contain producer 'pim mens' 
![image](https://cloud.githubusercontent.com/assets/2911176/10602360/88600d02-7763-11e5-8752-3755beaaffe9.png)

but shop is full of strawberries . . 
![image](https://cloud.githubusercontent.com/assets/2911176/10602365/95e7cc94-7763-11e5-9d28-b7b3750407a6.png)

relationships are ok - 

when I add pim mens into OC, the variants that were formerly in it reappear - they have not been cancelled out by removing the producer . . 

![image](https://cloud.githubusercontent.com/assets/2911176/10602433/04d0294e-7764-11e5-93a8-f8fd438c6b8b.png)

so i need to removethe variants from outgoing, and then remove the producer to get rid of them

BUT there is still one remaining 
![image](https://cloud.githubusercontent.com/assets/2911176/10602794/30290f64-7766-11e5-9094-e8acf9cc62d2.png)
although now, a bit later, I find that this has gone too . . mysterious

there are still two phantoms in the FC OC

**Comment by kirstenalarsen** [comment]
@RohanM - this one was in the order cycles fix list (http://community.openfoodnetwork.org/t/critical-order-cycle-updates/400) but apparently is still a problem. See Lisa's detailed explanation below 

As briefly texted about, last night we had almost 80 phantoms.  Here's the story of how i think that happened.

Due to Christmas holidays, we have a number of producers who are not supplying anything for next week.
Most of these are producer who we never normally touch... for whatever that is worth.  Leavain Bakery, Sprout Bakery, Cacao Girl are the main three that we noticed were showing on the shopfront even though the products were definitely un-ticked.  In addition were Russel & Marianne Pitt and Ray & Merilyn Clark.
When we had finished creating the order cycle it had the correct number (let's say 205/205... i can't remember the exact number) but when you re-opened the order cycle to edit it / check what the hell was going on - then it showed 285 / 205...
I solved it by deleting the distributor as a first step, saving and closing.
Opening to edit, deleting the offending producers entirely, as well as an any other producer who weren't supplying for that week (a spring clean if you will), saving and closing.
Opening to edit, adding a new distributor, and selecting all... saving and closing, then all seemed fine.  Stayed at 205/205 and appears to be holding steady.
